storybook/.storybook/decorators/ThemeProvider.decorator.tsx

```diff
@@ -0,0 +1,19 @@
+import { createCometTheme, MuiThemeProvider } from "@comet/admin";
+import { createTheme as createMuiTheme, CssBaseline } from "@mui/material";
+import { type Decorator } from "@storybook/react";
+
+export enum ThemeOptions {
+    Comet = "comet",
+    Mui = "mui",
```

Comment:
sometimes MUI Theme looks really broken. I am not sure if we want this option.

due to we already have a petrol Theme in our Comet DXP Library - maybe it would make more sense to showcase **Comet Default** and **Comet Petrol** Theme. 

Maybe it could also help developing component for different themes.

Reply:
I wasn't aware that we have a Comet Petrol Theme 😳 This isn't referenced anywhere in the code 🤔 

> sometimes MUI Theme looks really broken. I am not sure if we want this option.

Probably not. We may remove the theme switch altogether.

Reply:
As discussed, the other theme is just in Figma - but in my opinion probably worth showcasing another theme so we can check if components work proprietly with other comet themes.